### PR TITLE
Implement API

### PR DESCRIPTION
**Pull request by Kevin-Mena**
### In this Milestone,I Completed the following:
✅ Created a new component for displaying a quote
✅ Fetched data inside of a new component from the Quote API
✅ Show a loading state when loading data
✅ Show an error state if there is an error when loading data

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -10,6 +11,7 @@ const CalculatorApp = () => {
   return (
     <div>
       <div className="container-div">
+        <Quotes />
```
Comment (amejid) [inline src/components/calculator.js]: - [ ] Great job working on the `Quotes` component. :+1: Let's do a bit of refactoring to achieve the project requirements. According to the requirement, we are supposed to import the `Quotes` component into the main component. Kindly move it to the main component for a better structure. 🤝️

**Project requirement**
Screenshot from 2023-03-23 09-55-39

File: src/components/math-quotes.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      try {
+        const response = await fetch(url, {
+          headers: { 'X-Api-Key': API_KEY },
+          contentType: 'application/json',
+        });
+        const data = await response.json();
+        setData(data);
+      } catch (err) {
+        setIsError(true);
+        console.log(err);
```
Comment (amejid) [inline src/components/math-quotes.js]: - [ ] [OPTIONAL] Using `console.log` in client-side apps may lead to some security risks, one of which is sensitive information may be leaked. I highly suggest you remove it after you finished debugging your app. 😎️

**Review by amejid — CHANGES_REQUESTED** [review]
Hi @Kevin-Mena, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Gitflow :heavy_check_mark: 
- Great UI design :heavy_check_mark: 
- Data fetching working as expected :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/math-quotes.js (hunk 2 above)
Comment (Kevin-Mena) [reply src/components/math-quotes.js]: Hello 👋 @amejid,thank you for the reminder,here in are the implemented changes

File: src/components/calculator.js (hunk 1 above)
Comment (peterrobert) [reply src/components/calculator.js]: This change has been implemented well

File: src/components/math-quotes.js (hunk 2 above)
Comment (peterrobert) [reply src/components/math-quotes.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @Kevin-Mena, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**